api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/models/FunctionalModelTest.kt

```diff
@@ -63,20 +65,35 @@ internal class FunctionalModelTest {
     fun summary() {
         assertEquals("functional_model", correctTestModel.name)
 
-        val descriptions = correctTestModel.summary()
-        assertEquals(descriptions[0], "input_1(Input)                         [None, 28, 28, 1]         0")
-        assertEquals(descriptions[1], "conv2D_1(Conv2D)                       [None, 26, 26, 32]        320           input_1")
+        assertEquals(
+            ModelSummary(
+                type = "Functional",
+                name = "functional_model",
+                layersSummaries = listOf(
```

Comment:
could we add a yet one the same test, where all layers except dense_2 are frozen and we will have an alternative number of the frozenParamsCount, it improves the test coverage IMHO frozenParamsCount = 220 096, trainableParamsCount  = 2570

Reply:
I modified the existing test to contain both frozen and trainable parameters, is it ok?